graph-commons/src/test/scala/ch/datascience/http/client/IORestClientSpec.scala

```diff
@@ -228,8 +228,8 @@ class IORestClientSpec extends AnyWordSpec with ExternalServiceStubbing with Moc
     val client                = new TestRestClient(hostUrl, throttler, logger, Some(executionTimeRecorder))
 
     def verifyThrottling() = inSequence {
-      (throttler.acquire _).expects().returning(IO.unit)
-      (throttler.release _).expects().returning(IO.unit)
+      (() => throttler.acquire).expects().returning(IO.unit)
```

Comment:
Should we take the same approach here as in the other places, I mean to leave `throttler.acquire _` in the tests but add () to the acquire and release method?